quickwit-search/src/error.rs

```diff
@@ -41,6 +41,25 @@ pub enum SearchError {
     InvalidQuery(String),
 }
 
+impl SearchError {
+    fn convert_to_tonic_status_code(search_error: &SearchError) -> tonic::Code {
```

Comment:
There was probably code doing that somewhere, is it removed?

Reply:
Yes, I moved it here because of this need.

Reply:
awesome